Templates/Index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link href="https://allfont.ru/allfont.css?fonts=numans" rel="stylesheet" type="text/css">
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="../Static/CSS/Styles.css">
+</head>
+<body>
+	<header>
+		<img src="../Images/Logo.jpg" class="header_logo" alt="GumApp">
```

Comment:
@TitovIvan please make folder and files names to lowercase
`<img src="../images/Logo.jpg" class="header_logo" alt="GumApp">`

Reply:
done, earlier think only about folders and files names in local directory, not in code